src/open_inwoner/components/templatetags/link_tags.py

```diff
@@ -112,3 +112,15 @@ def get_classes():
     kwargs["href"] = get_href()
     kwargs["text"] = text
     return kwargs
+
+
+@register.filter
+def addnexturl(href, next_url):
+    """
+    Concatenates href & next_url.
+    """
+    if "?" in href:
+        href += f"&next={next_url}"
+        return href
+
+    return href + f"?next={next_url}"
```

Comment:
Unnecessary concatenation if we're already doing a template in the other half of the statement:

````python
return f"{href}?next={next_url}"
````